local_setup_scripts/win64/nuke.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash -e
```

Comment:
Did you mean to commit all these files? If so, then I think we need a README.md in the root of this folder that describes what they all do.

Reply:
yeah, sounds good